.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
 - glide install
 script:
 - go test . ./classifiers
-- $HOME/gopath/bin/goveralls -ignore examples/main.go -service=travis-ci -repotoken $COVERALLS_TOKEN
+- $HOME/gopath/bin/goveralls -ignore examples/example_app.go -service=travis-ci -repotoken $COVERALLS_TOKEN
```

Comment:
Something is broken here... Can you try:
```
install: go get github.com/mattn/goveralls 
script: goveralls -service travis-ci -repotoken $COVERALLS_TOKEN
```
I added the environment variable to travis. So you could also remove it from this file.